### PR TITLE
@broskoski: track fair artwork page

### PR DESCRIPTION
**Pull request by xtina-starr**
Related to this: https://github.com/artsy/force/pull/326

This makes the condition for tracking fair artworks less strict. This is done because urls like below were getting tracked:
https://www.artsy.net/artwork/alighiero-e-boetti-incontri-e-scontri-meetings-and-clashes?microsite=1&profile_id=artissima-2016&fair_id=artissima-2016&fair_name=Artissima%202016

But these were not: 
https://www.artsy.net/artwork/alighiero-e-boetti-incontri-e-scontri-meetings-and-clashes

They should both be tracked by sailthru.

### CODE REVIEW

File: apps/artwork/routes.coffee [hunk 1]
```diff
@@ -70,6 +69,7 @@ bootstrap = ->
       extend res.locals.helpers, helpers
       bootstrap res.locals.sd, data
       res.locals.sd.PARAMS = req.params
+      res.locals.sd.INCLUDE_SAILTHRU = data.artwork.fair?
```
Comment (xtina-starr) [inline apps/artwork/routes.coffee]: Wondering if I should safeguard the presents of artwork like so `data.artwork?.fair?`

**Comment by xtina-starr** [comment]
Seeing that this is red. I believe it's because I need to update the dependencies. I gotta run now but will address tomorrow.

**Comment by broskoski** [comment]
Not gonna block this if it works, but the changes looks like its more or less the same thing? Am I missing something?

**Comment by xtina-starr** [comment]
@broskoski yeah it kind of is. The original made sure microsite was in the url as a query. This PR doesn't require `microsite` to be in the url just that the artwork is in a fair.

**Comment by broskoski** [comment]
👍  got it, mergin!

**Comment by sperle** [comment]
@xtina-starr can you confirm with @rchglss when this is merged? I'll be OOO and she'll need to use a different template when she's staging the email tomorrow if it has. I don't think she can see tickets in force.

Thanks!

**Pull request merged**